ui/lib/ldap/addon/routes.js

```diff
@@ -9,6 +9,7 @@ export default buildRoutes(function () {
   this.route('overview');
   this.route('roles', function () {
     this.route('create');
+    this.route('subdirectory', { path: '/:type/subdirectory/*path_to_role' });
```

Comment:
Great naming and route structure.

Maybe add the comment here similar to the one included in KV [routes](https://github.com/hashicorp/vault/blob/main/ui/lib/kv/addon/routes.js#L11) noting why we're using wildcard vs dynamic. 

[docs for easy reference](https://guides.emberjs.com/v5.5.0/routing/defining-your-routes/#toc_wildcard--globbing-routes)